keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -123,51 +119,46 @@ type ServiceSpec struct {
 	...
 	...
 
-	// trafficPolicy denotes if the traffic for a Service should route
-	// to cluster-wide endpoints or node-local endpoints. "Cluster" routes traffic
-	// to a Service to all cluster-wide endpoints. "Topology" routes traffic based on
-	// topology hints. "PreferLocal" will route internal traffic to node-local endpoints
-	// if one exists, otherwise it will fallback to the same behavior as "Cluster".
-	// "Local" routes traffic to node-local endpoints only, traffic is dropped
-	// if no node-local endpoints are ready. When externalTrafficPolicy is "Cluster",
-	// traffic from external sources will be routed based on the trafficPolicy. When
-	// externalTrafficPolicy is "Local", trafficPolicy is ignored for traffic from
-	// external sources.
+	// InternalTrafficPolicy specifies if the cluster internal traffic
+	// should be routed to all endpoints or node-local endpoints only.
+	// "Cluster" routes internal traffic to a Service to all endpoints.
```

Comment:
This probably needs a bit more nuance since it's not necessarily all endpoints. I don't think the specifics need to block this PR though.

Reply:
This was just copied from what's in the API spec now, I agree we should re-evaluate the wording here.